bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -27,6 +27,9 @@ buildscript {
         jcenter()
         mavenCentral()
     }
+    dependencies {
+        "classpath"(group = "net.java.dev.jna", name = "jna", version = "4.5.2")
```

Comment:
Should I be using this weird `"classpath"` syntax?

Reply:
What does this do?

Reply:
It's making jna a dependency on the build itself, so we can use the ARCH constant on line 102

Reply:
Gotcha!